ui-tests/src/test/java/org/openmrs/reference/page/ManageLocationsPage.java

```diff
@@ -15,10 +15,24 @@
 
 public class ManageLocationsPage extends Page {
 
-    public static final By CONFIRM_PURGE_BUTTON = By.cssSelector("#adminui-purge-location-dialog > div.dialog-content > form > button.confirm.right");
+    protected static final String PAGE_URL = "/adminui/metadata/locations/manageLocations.page";
+    private static final String PARENT_LOCATION = "Pharmacy";
+    private static final By EDIT_LOCATION_LINK = By.cssSelector("i.icon-pencil.edit-action");
+    public static final By CONFIRM_PURGE_BUTTON = By.cssSelector("#adminui-purge-location-dialog button.confirm.right");
     public static final By CONFIRM_RETIRE_BUTTON = By.cssSelector("#adminui-retire-location-dialog > div.dialog-content > form > button.confirm.right");
-    protected static final String PAGE_URL = "adminui/metadata/locations/manageLocations.page";
+    private static final By LOCATION_TAG_BUTTON = By.cssSelector("#locationForm fieldset table tbody tr:first-child td:first-child input[type=\"checkbox\"]");
+    private static final By ADD_NEW_LOCATION_BUTTON = By.cssSelector("a.button");
+    private static final By LOCATION_NAME_FIELD = By.id("name-field");  
     private static final By ADD_EDIT_LOCATION = By.cssSelector("#content a[href='/openmrs/" + AddEditLocationPage.PAGE_URL + "']");
+    private static final By LOCATION_DESCRIPTION_FIELD = By.cssSelector("#description-field");
+    private static final By LOCATION_ADDRESS_FIELD_1 = By.cssSelector("#address1-field");
+    private static final By LOCATION_ADDRESS_FIELD_2 = By.cssSelector("#address2-field");
+    private static final By CITY_FIELD = By.cssSelector("#cityVillage-field");
```

Comment:
What happens when we change `#cityVillage-field` to `#testing123`?

Reply:
> What happens when we change `#cityVillage-field` to `#testing123`?

The CITY_FILED on the page can not be retrieved. Actually no element is found on the page with that element reference.

Reply:
Would the build fail?

Reply:
> Would the build fail?

No. 
The answer deserves a comment.

The build on distro will pass on this PR because that bug has no harm on distro module when building it !

But the E2E workflow test that will exercise the page functionality will not be able to get the element on the page hence its build will fail.

Reply:
Can you do it and i see what happens?

Reply:
> Can you do it and i see what happens?

Do you mean changing the value from `#cityVillage-field` to `#testing123` ? and you see the what comes out the build plan?

Reply:
> Can you do it and i see what happens?

@dkayiwa take a look at your request and see the build plan result of distro module.